openquakeplatform/templates/exposure/export.html

```diff
@@ -46,7 +46,7 @@
     <div id="national-spinner" >Loading ...<img id="download-button-spinner" src="{{ STATIC_URL }}img/ajax-loader.gif" /></div>
     {% verbatim %}
       <table ng-table="tableParams" show-filter="true" class="table ng-table-rowselected">
-        <tr ng-repeat="record in $data" ng-click="record.$selected = !record.$selected; changeSelection(record)"
+        <tr class="{{record.country_name}}{{record.num_studies}}" ng-repeat="record in $data" ng-click="record.$selected = !record.$selected; changeSelection(record)"
```

Comment:
Could we use a different attribute to match `tr` ?

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/aab0358c7d2def982eebfb9b57a8158e483544d9